components/provider_cldr/src/transform/dates.rs

```diff
@@ -161,10 +161,11 @@ impl From<&cldr_json::DateTimeFormats> for gregory::patterns::DateTimeFormatsV1
                     };
 
                     if !variant_parts.is_empty() {
-                        unimplemented!(
+                        eprintln!(
```

Comment:
Nit (optional): Make this print only in debug mode.  Maybe `dbg!` ?

Reply:
I'd prefer to always keep it, since we don't really exercise this code in debug mode.